### ISSUE TITLE
Chat options

### ISSUE THREAD

**Issue by Dune-jr** [teeworlds/teeworlds#2293]
It seems like there is some demand for chat options, at least

- select chat categories to disable sounds for: https://github.com/teeworlds/teeworlds/pull/2277
- select chat categories to disable entirely: https://github.com/teeworlds/teeworlds/pull/2289

Those options must be consistent with respect to each other (you don't want sounds for invisible chat messages).
Should these options be grouped together somehow?

**Comment by jxsl13** [comment]
Check out the chat sound options as well, which could be used in correspondance to tue visibility options.( pr already exists)

**Comment by oy** [comment]
Yeah should be consistent and not to compilcated. To group them together seems like a good idea 👍  Like either disable a category completly or just disable its sound?

**Comment by SoulyVEVO** [comment]
Maybe we could have some more chat options like using the old chat sound from previous versions which i know some players liked. And also more options to disable/enable different types of chat sounds including the `say_self` sound.
Here is how it could look like in game:
![coolimage](https://user-images.githubusercontent.com/61762251/105167316-d96bde80-5b18-11eb-8399-ad5efb21ee20.png)

**Comment by Robyt3** [comment]
@SoulyVEVO Replacing sounds is what #34 (the `storage.cfg`) is for. Just add a new folder as highest priority that replaces the sounds you want.

**Comment by jxsl13** [comment]
highest priority = at the too :0

**Comment by SoulyVEVO** [comment]
@Robyt3 Wouldn't that replace/disable the sound completely? Thats not really what i want, i want to mute specific sound but **only** for chat, some mods actually use those sounds for different things and i want to hear those.
